src/plugins/mer/merbuildsettingswidget.ui

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>Mer::Internal::MerBuildSettingsWidget</class>
+ <widget class="QWidget" name="Mer::Internal::MerBuildSettingsWidget">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>335</width>
+    <height>23</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string/>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout">
+   <property name="leftMargin">
+    <number>0</number>
+   </property>
+   <property name="topMargin">
+    <number>0</number>
+   </property>
+   <property name="rightMargin">
+    <number>0</number>
+   </property>
+   <property name="bottomMargin">
+    <number>0</number>
+   </property>
+   <item>
+    <widget class="QCheckBox" name="m_importVariablesCheckbox">
+     <property name="sizePolicy">
+      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
+       <horstretch>0</horstretch>
+       <verstretch>0</verstretch>
+      </sizepolicy>
+     </property>
+     <property name="text">
+      <string>Import variables from Sailfish OS build engine</string>
```

Comment:
I am afraid it will be very hard to say something meaningful here. And I think is it a clear sign that what we do here is really a hack :) And it is still unclear if disabling this is ever going to be needed.

With respect to these I would prefer keeping the whole widget hidden by default and add another two checkboxes to Options > Sailfish OS > General: one to enable this UI, and another one to change the default from true to false.

Another option would be to fully omit this UI and add just the global option to disable this feature.

Reply:
`merbuildsettingswidget.ui` removed. Checkbox now in Options > Sailfish OS > General.